vsintegration/ProjectTemplates/ConsoleProject/Template/ConsoleApplication.fsproj

```diff
@@ -70,12 +70,12 @@
   <Choose>
     <When Condition="'$(VisualStudioVersion)' == '11.0'">
       <PropertyGroup Condition="Exists('$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.0\Framework\v4.0\Microsoft.FSharp.Targets')">
-        <FSharpTargetsPath>$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.0\Framework\v4.0\Microsoft.FSharp.Targets</FSharpTargetsPath>
+        <FSharpTargetsPath Condition="'$(FSharpTargetsPath)'==''">$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.0\Framework\v4.0\Microsoft.FSharp.Targets</FSharpTargetsPath>
```

Comment:
It would be fair to say that we don't do that spacing anywhere near consistently.  So it's fine as it is.  Checkout the line immediately above :-).